libkarel.py

```diff
@@ -337,6 +353,21 @@ def instrucciones(self):
         """Un diccionario con el número de instrucciones que karel ejecutó"""
         return self.__instrucciones
 
+    @property
+    def longitud_stack(self):
+        """El tamanio maximo que puede tener el stack"""
+        return self.__longitud_stack
+
+    def limiteComando(self, comando):
+        """Regresa el número máximo de veces que se puede usar un comando
+
+        Si no hay límite, regresa false
+        """
+        if comando not in self.__limites:
+            return false
```

Comment:
```suggestion
            return None
```

es mejor regresar `None` para que tenga el mismo "tipo".